frontend/src/v5/helpers/colors.helper.ts

```diff
@@ -89,13 +89,20 @@ export const rgbGroupColorToHex = ({ opacity, color }: RgbGroupColor): HexGroupC
 
```

Comment:
I can't set "no color" in ticket groups

https://github.com/user-attachments/assets/57789447-6729-4bf3-b588-394a22350adc